.github/workflows/wheels.yml

```diff
@@ -214,14 +223,21 @@ jobs:
           name: artifact
           path: dist
 
+      # Upload to PyPI
       - uses: pypa/gh-action-pypi-publish@release/v1
+        if: github.event_name == 'release' || (github.event_name == 'workflow_dispatch' && github.event.inputs.upload_dest == 'PyPI')
         with:
           # PyPI does not allow replacing a file. Without this flag the entire action fails if even a single duplicate exists.
           skip-existing: true
           verbose: true
-          # Real PyPI:
-          password: ${{ secrets.PYPI_TOKEN }}
+          password: ${{ secrets.PYPI_API_TOKEN }}
```

Comment:
This repo currently only has PYPI_TOKEN secret. I just invited you to PYPI and this repo; let me know if you don't have correct permissions to handle secrets.

Reply:
Ha, good catch! Spent too much time double checking the `if`s and blindly copy-pasted this part.